yt_dlp/extractor/nekohacker.py

```diff
@@ -0,0 +1,125 @@
+    _TESTS = [{
+        'url': 'https://nekohacker.com/nekoverse/',
+        'info_dict': {
+            'id': 'nekoverse',
+            'title': 'Nekoverse',
+        },
+        'playlist': [
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2022/11/01-Spaceship.mp3',
+                'md5': '44223701ebedba0467ebda4cc07fb3aa',
+                'info_dict': {
+                    'id': '01-Spaceship',
+                    'ext': 'mp3',
+                    'title': 'Spaceship',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2022/11/Nekoverse_Artwork-1024x1024.jpg',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'upload_date': '20221101'
+                }
+            },
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2022/11/02-City-Runner.mp3',
+                'md5': '8f853c71719389d32bbbd3f1a87b3f08',
+                'info_dict': {
+                    'id': '02-City-Runner',
+                    'ext': 'mp3',
+                    'title': 'City Runner',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2022/11/Nekoverse_Artwork-1024x1024.jpg',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'upload_date': '20221101'
+                }
+            },
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2022/11/03-Nature-Talk.mp3',
+                'md5': '5a8a8ae852720cee4c0ac95c7d1a7450',
+                'info_dict': {
+                    'id': '03-Nature-Talk',
+                    'ext': 'mp3',
+                    'title': 'Nature Talk',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2022/11/Nekoverse_Artwork-1024x1024.jpg',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'upload_date': '20221101'
+                }
+            },
+            {
+                'url': 'https://nekohacker.com/wp-content/uploads/2022/11/04-Crystal-World.mp3',
+                'md5': 'd8e59a48061764e50d92386a294abd50',
+                'info_dict': {
+                    'id': '04-Crystal-World',
+                    'ext': 'mp3',
+                    'title': 'Crystal World',
+                    'thumbnail': 'https://nekohacker.com/wp-content/uploads/2022/11/Nekoverse_Artwork-1024x1024.jpg',
+                    'vcodec': 'none',
+                    'acodec': 'mp3',
+                    'upload_date': '20221101'
+                }
+            }
+        ]
+    }]
+
+    def _real_extract(self, url):
+        playlist_id = self._match_id(url)
+
+        webpage = self._download_webpage(url, playlist_id)
+
+        webpage = get_element_by_class('playlist', webpage)
+
+        if webpage is None:
+            raise ExtractorError('no playlist element found - likely not a album', expected=True)
```

Comment:
For readability, just use a new variable instead of redefining `webpage` since then the name is no longer accurate
```suggestion
        playlist_id = self._match_id(url)
        webpage = self._download_webpage(url, playlist_id)
        playlist = get_element_by_class('playlist', webpage)
        if playlist is None:
            raise ExtractorError('No playlist element found; this URL is likely not for an album', expected=True)
```

Reply:
done with 50e62aa18f1da700ecc23f39484ca028c7bd76d9